### PR TITLE
Stamps 2 : the mergening

### PR DESCRIPTION
**Pull request by Lyacs**
## About the PR

New PR after messing up merging upstream on my previous one.

New stamps for RP purposes (will be PR on MRP later) and for mappers in the future. Groundwork for a future stamp PR for antags.

It adds stamps from corporations and governments. 

## Why / Balance

- Deepens lawyer (and NTR) roleplay interaction without changing the RP
- Corporations interact between each-other (on tg lore at least), despite funding the syndicate. Sending corporate lawyers on other station is normal.
- Makes more logical that some lawyer will try to work with prisoners rather than sec without changing anything really, corpo lawyer just working pro-bono to annoy NT sec off. Whereas it's weird that currently a lot of lawyer are hired by NT (so NT loyal) but still going more 90% against sec by doing pro-bono for the criminals and syndies (even if they aren't antag themselves)
- A 20h job time is required to ensure only people liking a minimum roleplay and paperwork will use them, so that have enough braincell to understand what a lawyer is,
- In addition a small description is added to each of the loadout stamps to make explicit these stamps are just for roleplaying corpo lawyers.
## Technical details

Changes loadout, added new stamps, now basic lawyer stamps is in the loadout area and no longer a starting gear (yet they will be force to have it before the 20h mark)

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Added new stamps for mappers and to the lawyer loadout for more roleplay interactions. They are NOT a self-antag permit, lawyers respect Space Law  >.>

### CODE REVIEW

**Comment by Lyacs** [comment]
Will fix the typo tmr

File: Resources/Prototypes/Loadouts/role_loadouts.yml [hunk 1]
```diff
@@ -106,6 +106,7 @@
   - Glasses
   - Survival
   - Trinkets
+  - LawyerStamps # Goobstation
```
[the remark this answers is not in the document]
Comment (Lyacs) [reply Resources/Prototypes/Loadouts/role_loadouts.yml:109]: already like this

**Comment by Lyacs** [comment]
Forgot to say that it is working without any problems despite the last check

**Comment by Aidenkrz** [comment]
yeah basically everyone on the staff team agrees it makes no sense for the lawyers to have enemy corp stamps, and NT wouldn't hire enemy lawyers, if you want to make this a syndie item/stick it in with the bureaucracy bundle go for it in a different pr.

**Pull request closed without merging**